set.mm

```diff
@@ -76238,14 +76236,14 @@ currently used conventions for such cases (see ~ cbvmpt2x , ~ ovmpt2x and
     $( Ordinal multiplication with 1.  Proposition 8.18(2) of [TakeutiZaring]
        p. 63.  (Contributed by NM, 3-Aug-2004.) $)
     om1r $p |- ( A e. On -> ( 1o .o A ) = A ) $=
-      ( vx vy c1o cv comu co wceq c0 oveq2 id eqeq12d con0 wcel 1on wa coa mpan
-      ciun cvv csuc ax-mp omsuc oveq1 sylan9eq oa1suc adantr eqtrd ex wral wlim
-      om0 cuni iuneq2 uniiun syl6eqr vex omlim limuni syl5ibr tfinds ) DBEZFGZV
-      BHZDIFGZIHZDCEZFGZVGHZDVGUAZFGZVJHZDAFGZAHBCAVBIHZVCVEVBIVBIDFJVNKLVBVGHZ
-      VCVHVBVGVBVGDFJVOKLVBVJHZVCVKVBVJVBVJDFJVPKLVBAHZVCVMVBAVBADFJVQKLDMNZVFO
-      DULUBVGMNZVIVLVSVIPVKVGDQGZVJVSVIVKVHDQGZVTVRVSVKWAHODVGUCRVHVGDQUDUEVSVT
-      VJHVIVGUFUGUHUIVICVBUJZVDVBUKZCVBVHSZVBUMZHWBWDCVBVGSWECVBVHVGUNCVBUOUPWC
-      VCWDVBWEVBTNZWCVCWDHZBUQVRWFWCPWGOCDVBTURRRVBUSLUTVA $.
+      ( vx vy c1o cv comu co wceq c0 oveq2 id eqeq12d con0 wcel wa coa 1on mpan
+      ciun cvv csuc om0x omsuc oveq1 sylan9eq oa1suc adantr ex wral wlim iuneq2
```

Comment:
om0x is discouraged?

Reply:
Good catch, I didn't think about this circumstance. What happened is that it didn't have a proper discouraged label when I forked the repository, but it was edited afterwards.

I'll give a look at all pull requests after my fork and see all theorems that were discouraged in the meantime (they are not that many so I don't think that will take long), then I'll remove all the occurrences of those theorems that appear in this commits. It's a bit annoying, but not terrible. 

Definitely when a discouraged label will be added in the future I would very much appreciate being pinged.

I'll get back with more info, I need some time for this.

Reply:
Now that I think about it, shouldn't the CI pipeline fail? In discouraged there is `New usage of "om0x" is discouraged (0 uses).` But that would be incorrect with this commit, so why are all checks passing without any issues?

Reply:
I think I understand why it didn't fail, the discouraged label was added very recently in this commit https://github.com/metamath/set.mm/pull/3156/files. I made the pull request before that commit was merged, so the pipeline was executed when `om0x` wasn't still discouraged and so every check passed. If I trigger the CI pipeline again, it should fail then.

Reply:
Ok, I checked back through PRs and all discouraged additions that I found (besides `om0x`) were theorems that were added after my fork (so they couldn't be used by my minimization process). So it seems to me that `om0x` is the only oddity here.

I would still like to trigger the pipeline to have evidence of that tho. How can I do that?

Reply:
Change to tab 'Checks', and use the button 'Re-run jobs'

Reply:
I don't understand, the correct label should be `New usage of "om0x" is discouraged (4 uses).` since in this pull request `om0x` was used in 4 thorems for minimization. But in the currently discouraged file in [metamath develop](https://github.com/metamath/set.mm) it's `New usage of "om0x" is discouraged (0 uses).` and I didn't touch that file in this PR. Shouldn't the pipeline complain about that?

Reply:
It may help to merge the 'develop' branch into your branch in this PR.  This will update the discouraged file to the latest state, including an entry for the om0x theorem, and a list of all allowed users (should be empty). My copy contains a line

New usage of "om0x" is discouraged (0 uses).

If your copy of discouraged in your branch has this line, too, then the CI process reads it, and my assumption is that it will fail.  I don't know much about the details of its implementation, but I am under the impression that it only checks that a branch is inherently consistent, and does not take into account the diff to 'develop'.

Reply:
Ok, that seems to be the correct diagnosis